metadata-ingestion/src/datahub/cli/delete_cli.py

```diff
@@ -199,7 +199,7 @@ def delete(
             )
             remove_references = click.confirm("Do you want to delete these references?")
 
-        if remove_references:
+        if force or remove_references:
```

Comment:
Why does force also remove references? I think that all deletes should remove references but not sure if thats communicated in docs

Reply:
The `-f` option in CLI is used to bypass checks for CI. If someone was trying to delete many URNs then this will cause them to need to do `yes` for each one of them. 

I was trying to do this for 90 URNs
`datahub delete --urn URN -f`

For almost more than half of them of them I had to press `y` to confirm. Usually all CLIs tools bypass all confirmations when `-f` is used. I am making the behaviour consistent with that.